.ci/jobs/defaults.yml

```diff
@@ -51,7 +51,6 @@
         timeout: '15'
         use-author: true
         wipe-workspace: 'True'
-    periodic-folder-trigger: 1w
```

Comment:
This seems unrelated to the purpose of this PR. If this was intended, perhaps move it to a different PR?